llvm/lib/CodeGen/FinalizeISel.cpp

```diff
@@ -69,6 +72,15 @@ bool FinalizeISel::runOnMachineFunction(MachineFunction &MF) {
     }
   }
 
+  for (auto &MBB : MF)
```

Comment:
I don't really think it's worth adding an extra loop for it. It doesn't seem like it would be too bad to require custom inserters to set this. It's more OK if we could catch this with the verifier

Reply:
OK - patch updated. I now actually found out there were cases of emitting calls even after finalize isel (PPCTLSDynamicCall.cpp), which made it obvious that this should be verified as late as possible. Seems there is no real need to employ the verifier - a single assert in PEI does the job, I think.